### DISCUSSION TITLE
npmjs listing package as no longer supported

### DISCUSSION THREAD

**Discussion by jimpanzeeee** [golevelup/nestjs#598] [Q&A] [answered]
Is this true? I can see the last update was 3 weeks ago and some activity in the forums so this seems wrong?
https://www.npmjs.com/package/@golevelup/nestjs-testing

**Comment by WonderPanda** [comment, marked as answer]
The package was renamed and the one you linked is no longer receiving updates. The correct package to use which replaced it is [@golevelup/ts-jest](https://www.npmjs.com/package/@golevelup/ts-jest)
